src/Microsoft.MobileBlazorBindings.Core/NativeControlComponentBase.cs

```diff
@@ -26,10 +27,33 @@ protected override void BuildRenderTree(RenderTreeBuilder builder)
             builder.OpenElement(0, GetType().FullName);
             RenderAttributes(new AttributesBuilder(builder));
 
+            var index = 2;
```

Comment:
I tried to preserve same behavior as before (see line 56), but I have no idea why it's 2 here, and not 1.